src/PageContent/RequestPickup/PickupSummary.js

```diff
@@ -46,8 +46,16 @@ class PickupSummary extends React.Component {
                                     <p>email</p>
                                 </div>
                             </div>
-                            {/* TODO: Pass in address into map */}
-                            <Map></Map>
+                            {/* concat address fields into a string */}
+                            <Map 
+                                address={Object.keys(this.props.address).map(key => this.props.address[key]).join(' ')}
```

Comment:
`address` fetched from the db will always be in the following format:
```
address: {
                street1: '1201 NE Campus Pkwy',
                street2: '',
                city: 'Seattle',
                state: 'WA',
                zipcode: 98105,
                officeNo: '220'
            }
```
So I think `Map` should take in this format and do this `join` operation before using the address, so that we don't need to do this operation everywhere we use `Map`.